website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -0,0 +1,683 @@
+information of the file in the table. By doing this we do simplify reconciliation
+with Cumulus but, then the only catalog of all the versions available for a file
+would reside in the glacier S3 bucket. The implication of this is there is no other
+record of the various versions of the file. Also, we should need to query the S3
+bucket if the data manager wanted to inquire what versions were available for a
+file in order to restore the proper version back or to identify versions that may
+be eligible for deletions based on policy. If we merge the objects, we may want to
+capture the total versions available for the file.
+
+
+### Return Metadata Needed by Cumulus
+
+At a minimum, the following metadata should be returned back to Cumulus upon a
+successful query for reconciliation return from ORCA. The data below is what
+is needed for reconciliation comparison and general metadata that may be needed
+by the end user to perform analysis on discrepencies.
+
+- Provider ID (providerId)
+- Collection ID (collectionId)
+- Granule ID (granuleId)
+- Granule Date - TBD dependent on what is used for date range comparison ideally acquisition
+- Granule Ingest Date (ORCA) - may also be referred to as a Create Date.
+- Granule Last Update (ORCA)
+- File Name
+- ORCA Location - archive bucket + archive path
+- File Size
+- File Checksum
+- File Checksum Type
+- File Version (always send back latest version to compare)
+
+A rough structure may look like the following below. In order to simplify
+comparisons, only the latest file version information will be returned. Enhancements
+we will need consider later include items like updating the latest flag in the
+database and glacier on a file version when doing a recovery.
+
+```json
+[
+  {
+    "providerId": "lpdaac",
+    "collections": [
+      {
+        "collectionId": "MOD14A1__061",
+        "granules": [
+          {
+            "granuleId": "MOD14A1.061.A23V45.2020235",
+            "date": "TBD",
+            "createDate": "2020-01-01T23:00:00+0",
+            "lasstUpdate": "2020-01-01T23:00:00+0",
+            "files": [
+              {
+                "fileName": "MOD14A1.061.A23V45.2020235.2020240145621.hdf",
+                "orcaLocation": "s3://orca-archive/MOD14A1/061/...",
+                "fileSize": 100934568723,
+                "fileChecksum": "ACFH325128030192834127347",
+                "fileChecksumType": "SHA-256",
+                "fileVersion": "VXCDEG902",
+                "orcaEtag": "YXC432BGT789"
+              },
+              ...
+            ]
+          },
+          ...
+        ]
+      },
+      ...
+    ]
+  },
+  ...
+]
+```
+
+Some items we will need to think about include managing paging (start/stop) for
+large returns and limiting size of the message across the wire.
+
+
+### Potential ORCA Reconciliation Data Reporting Architecture
+
+The following provides a rough high level architecture for reporting the ORCA
```

Comment:
The following provides a rough, high-level architecture for reporting the ORCA

Reply:
fixed